src/core/config/openthread-core-config-check.h

```diff
@@ -487,8 +487,11 @@
 #endif
 
 #ifdef OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT
-#error \
-    "OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT (in seconds) was replaced by OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT_MILLIS (in milliseconds)"
+#error "OPENTHREAD_CONFIG_COAP_ACK_TIMEOUT (in seconds) was removed."
```

Comment:
Should give some clarification on why these configuration are removed? Why they are not useful anymore?